ansible/axelar-datadog.yml

```diff
@@ -0,0 +1,52 @@
+- hosts: validator_axelar_mainnet
+  become: yes
+  become_method: sudo
+  roles:
+    - { role: Datadog.datadog, become: yes }
+  vars_files:
+    - /Volumes/Keybase/team/figureops/ansible/datadog_vars
+  vars:
+    datadog_agent_major_version: "7"
+    datadog_additional_groups:
+      - systemd-journal
+    datadog_config:
+      logs_enabled: true
+      tags:
+        - "env:{{ env }}"
+        - "chain_id:{{ chain_id }}"
+        - "component:blockchain"
+        - "nodename: {{ inventory_hostname }}"
```

Comment:
I would need to dig more into whether tags get stripped of whitespace or not. not sure if this would cause any funkiness down the line